sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc

```diff
@@ -322,11 +322,126 @@ q.submit([&](sycl::handler& cgh) {
 });
 q.wait();
 ```
+
+=== Load/Store Using Annotated Pointers
```

Comment:
I think we could define these load/store variants with `annotated_ptr` in the "oneapi" spec because there could be some "oneapi" properties that make sense.  For example, there is another conversation where we think `alignment` would be useful.  If we do that, this "intel" spec can document the "intel" properties that can also be used with these APIs.

Reply:
@gmlueck, that makes sense. 
But are you suggesting adding these at the end of the "oneapi" document or blend them with the existing ones under the same section?  
I am hesitant about adding/mixing these new APIs with the ones we already have, that are somewhat stable. In the event we decide to promote the stable parts, there should be clear distinction between the stable ones and the new added ones that should also go through the cycle of implementation and stable usage.
Do you think there is a good way to address this?

Reply:
I was thinking that they would be documented in the same section as the other load/store APIs.

When it comes time to promote the matrix extension out of "experimental", we can decide whether these APIs are not yet ready.  If that is that case, we can remove them from the "supported" extension and leave them in the "experimental" one.